docs/ballooning.md

```diff
@@ -61,11 +60,19 @@ memory usage, they can be broken by a compromised driver in the guest. The
 balloon device operates on a best effort model and users should always ensure
 the host is prepared to handle a situation in which the Firecracker process
 uses all of the memory it was given at boot even if the balloon was used to
-restrict the amount of memory available to the guest.
+restrict the amount of memory available to the guest. It is also the users'
+responsibility to monitor the memory consumption of the VM and, in case
+unexpected increases in memory usage are observed, we recommend the following
+options:
+* through the snapshot capabilities, migrate the VM to a machine with higher
+memory availability;
+* kill the Firecracker process that exceeds memory restrictions;
+* enable swap with a sufficient amount of memory to handle the memory demand;
```

Comment:
I would also mention the cost of doing so.

Reply:
Done.